pkg/rtc/participant.go

```diff
@@ -597,9 +597,6 @@ func (p *ParticipantImpl) SetMigrateState(s types.MigrateState) {
 	var pendingOffer *webrtc.SessionDescription
 	p.migrateState.Store(s)
 	if s == types.MigrateStateSync {
-		if !p.hasPendingMigratedTrack() {
-			p.migrateState.Store(types.MigrateStateComplete)
-		}
```

Comment:
Will this now be handled in `handleTrackPublished`?

Reply:
Both primary peer connection state change and handleTrackPublished will handle the migrate state